sbc_translation/wizards/reply_to_comments.py

```diff
@@ -0,0 +1,14 @@
+from odoo import models, fields, api
+
+
+class ReplyToComments(models.TransientModel):
+    _name = "reply.to.comments"
```

Comment:
I would suggest a name that identifies better what it is. (Note: this is to change everywhere)
```suggestion
    _name = "sbc.reply.to.comments.wizard"
```